### ISSUE TITLE
colored droplets on the result image

### ISSUE THREAD

**Issue by sangjun04** [LynnHo/CycleGAN-Tensorflow-Simple#32]
colored dots? droplets? are appearing on the result image. 
I am really curious on what may caused this issue.

Any tips or idea on how to reduce such noise?

**Comment by LynnHo** [comment]
@sangjun04 Maybe the droplets are caused by the architecture? I am not sure. You can refer to StyleGAN v2 paper which solves the droplet problem caused by instance normalization.

**Comment by Zixin-Yong** [comment]
I found this effect in earlier epochs. Try to train the model longer to see if it goes away.
